timemachine/ff/handlers/utils.py

```diff
@@ -67,3 +73,57 @@ def match_smirks(mol, smirks):
         matches.append(tuple(mas))
 
     return matches
+
+
+def convert_to_oe(mol):
+    """Convert an ROMol into an OEMol"""
+
+    # imported here for optional dependency
+    from openeye import oechem
+
+    mb = Chem.MolToMolBlock(mol)
+    ims = oechem.oemolistream()
+    ims.SetFormat(oechem.OEFormat_SDF)
+    ims.openstring(mb)
+
+    for buf_mol in ims.GetOEMols():
+        oemol = oechem.OEMol(buf_mol)
+    ims.close()
+    return oemol
+
+
+def get_symmetry_classes(rdmol: Chem.Mol) -> NDArray:
+    """Convenient wrapper for [atom.GetSymmetryClass() for atom in oemol]
+
+    (Accepts rdmol instead of oemol, and renumbers classes to be contiguous from 0 to n_classes.)
+    """
+
+    # imported here for optional dependency
+    from openeye import oechem
+
+    oemol = convert_to_oe(rdmol)
+    oechem.OEPerceiveSymmetry(oemol)
```

Comment:
1) is this stereo-aware?
2) bool OEPerceiveSymmetry(OEMolBase &, bool includeH=true, bool automorph=false), what does automorph do here? API docs unfortunately are quite sparse. @bp-kelley @badisa might know...